From 673da5e4f6c9b6c54bb915aa07fc6ea726508327 Mon Sep 17 00:00:00 2001
From: Ludovic Desroches <ludovic.desroches@atmel.com>
Date: Thu, 24 Jan 2013 16:13:39 +0100
Subject: [PATCH 295/319] video: atmel_hlcdfb: don't wait when going to
 suspend

For an unknown reason polling the end of dma transfer before going to suspend
leads to an infinite loop on sama5d3x.

Signed-off-by: Ludovic Desroches <ludovic.desroches@atmel.com>
---
 drivers/video/atmel_hlcdfb.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/atmel_hlcdfb.c b/drivers/video/atmel_hlcdfb.c
index e648ffc..3897c9b 100644
--- a/drivers/video/atmel_hlcdfb.c
+++ b/drivers/video/atmel_hlcdfb.c
@@ -436,7 +436,7 @@ static int atmel_hlcdfb_suspend(struct platform_device *pdev, pm_message_t mesg)
 	if (sinfo->atmel_lcdfb_power_control)
 		sinfo->atmel_lcdfb_power_control(0);
 
-	atmel_hlcdfb_stop(sinfo, 0);
+	atmel_hlcdfb_stop(sinfo, ATMEL_LCDC_STOP_NOWAIT);
 	atmel_lcdfb_stop_clock(sinfo);
 
 	return 0;
-- 
1.7.10.4

